dashboard/src/components/topbar/PublicTopBar.tsx

```diff
@@ -31,23 +32,23 @@ export default function PublicTopBar() {
   return (
     <header className='bg-background/95 supports-[backdrop-filter]:bg-background/60 sticky top-0 z-50 w-full border-b backdrop-blur'>
       <div className='mx-auto max-w-7xl px-8'>
-        <div className='flex h-14 items-center justify-between'>
+        <div className='flex h-(--topbar-height) items-center justify-between'>
           <div className='flex items-center space-x-2'>
             <Link href='/' className='flex items-center space-x-2' onClick={closeMobileMenu}>
               <Logo variant='icon' showText textSize='md' priority />
             </Link>
           </div>
 
           <nav className='hidden items-center space-x-6 md:flex'>
-            <a
+            <ExternalLink
               href='/docs'
               title='Complete Betterlytics Documentation'
               className='text-muted-foreground hover:text-foreground text-sm font-medium transition-colors'
             >
               Documentation
-            </a>
+            </ExternalLink>
             <Link
-              href='#pricing'
+              href='/#pricing'
```

Comment:
Question: Why change this? 😄

Reply:
That makes sense.
However, it seems to introduce a new bug on the landing page.
If the landing page is already loaded, and I click on the "pricing", it needs to fetch almost 1mb of data + do a full navigation before being able to scroll down.
So it seems to be doing a full re-fetch of something, before it can even scroll - it potentially has to re-fetch the entire landing page. This feels a little clunky instead of just scrolling down to already loaded content, if that makes sense 😄 

<img width="1910" height="921" alt="image" src="https://github.com/user-attachments/assets/6b539f3f-f1bb-42f9-9980-448079a8116f" />

Reply:
Nevermind, this also happens on `main`. It must be a dev thing